### ISSUE TITLE
Alt text for Lightbox

### ISSUE THREAD

**Issue by aksenovaa** [benjaminkott/bootstrap_package#1196]
# Alt text for Lightbox Feature Request

Lightbox popup image code missing alt text for image. Can it be added? I didn’t figure out how, I need to edit the code for LightboxViewHelper.php

**Comment by benjaminkott** [comment]
is resoved in master https://github.com/benjaminkott/bootstrap_package/blob/master/Classes/ViewHelpers/Link/LightboxViewHelper.php#L37
